sources/postgres/data.go

```diff
@@ -186,13 +189,16 @@ func convInt64(val string) (int64, error) {
 // into a string representing a valid Spanner numeric.
 // Ideally we would just return a *big.Rat, but spanner.Mutation
 // doesn't currently support use of *big.Rat.
-// TODO: return *big.Rat when client library supports it.
```

Comment:
The new code returns big.Rat for regularly spanner NUMERIC types --- does that mean spanner.Mutation now supports big.Rat?

If so, thanks for resolving the TODO.
Can you also clean up the comment -- we don't need the "Ideally we would just return a *big.Rat, but spanner.Mutation doesn't currently support use of *big.Rat.".
Also, please update the convNumeric comment - it no longer returns a string.

Reply:
Yes, client library does support big.Rat now, but I ran into trouble updating the test cases to expect big.Rat instead of string. I have thus reverted back cleaning up this TODO for now.